### ISSUE TITLE
npm WARN deprecated jade@1.9.2: Jade has been renamed to pug, please install the latest version of pug instead of jade npm ERR! Windows_NT 10.0.14393

### ISSUE THREAD

**Issue by abobakr11** [GoogleCloudPlatform/nodejs-getting-started#122]
_I working  on nodeJS project after install express I wrote in Git bash npm install , that error appear to me_ 
```
`npm WARN deprecated jade@1.9.2: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm ERR! Windows_NT 10.0.14393

```
_now 
I need to understand why this error appear and how can i solve thise pug`_

**Comment by ace-n** [comment]
Hi @abobakr11,

This was fixed in #119. Are you using an outdated version, or did you mean to post about this on another repo?

**Comment by ace-n** [comment]
Closing this - @abobakr11 please reopen if this is still an issue for you. Thanks! 😄
